smoke-test/src/test/java/org/opennms/smoketest/sentinel/SentinelThrehsoldingIT.java

```diff
@@ -0,0 +1,213 @@
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with OpenNMS(R).  If not, see:
+ *      http://www.gnu.org/licenses/
+ *
+ * For more information contact:
+ *     OpenNMS(R) Licensing <license@opennms.org>
+ *     http://www.opennms.org/
+ *     http://www.opennms.com/
+ *******************************************************************************/
+
+package org.opennms.smoketest.sentinel;
+
+import static org.awaitility.Awaitility.await;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.hasItem;
+import static org.hamcrest.Matchers.not;
+import static org.opennms.netmgt.events.api.EventConstants.HIGH_THRESHOLD_EVENT_UEI;
+
+import java.io.IOException;
+import java.net.InetSocketAddress;
+import java.util.AbstractMap;
+import java.util.Date;
+import java.util.Map;
+import java.util.Random;
+import java.util.Set;
+import java.util.concurrent.TimeUnit;
+import java.util.stream.Collectors;
+
+import org.junit.ClassRule;
+import org.junit.Ignore;
+import org.junit.Test;
+import org.opennms.netmgt.model.OnmsAlarmCollection;
+import org.opennms.netmgt.model.OnmsNode;
+import org.opennms.netmgt.model.OnmsSeverity;
+import org.opennms.netmgt.telemetry.protocols.jti.adapter.proto.Port;
+import org.opennms.netmgt.telemetry.protocols.jti.adapter.proto.TelemetryTop;
+import org.opennms.smoketest.stacks.BlobStoreStrategy;
+import org.opennms.smoketest.stacks.IpcStrategy;
+import org.opennms.smoketest.stacks.JsonStoreStrategy;
+import org.opennms.smoketest.stacks.NetworkProtocol;
+import org.opennms.smoketest.stacks.OpenNMSProfile;
+import org.opennms.smoketest.stacks.OpenNMSStack;
+import org.opennms.smoketest.stacks.StackModel;
+import org.opennms.smoketest.telemetry.Packet;
+import org.opennms.smoketest.utils.KarafShell;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class SentinelThrehsoldingIT {
+    private static final Logger LOG = LoggerFactory.getLogger(SentinelThrehsoldingIT.class);
+    private static final String NODE_IP = "192.168.1.1";
+    private static final String INTERFACE_ID = "eth0_system_test";
+
+    @ClassRule
+    public static final OpenNMSStack stack = OpenNMSStack.withModel(StackModel.newBuilder()
+            .withOpenNMS(OpenNMSProfile.newBuilder()
+                    .withFile("sentinel-thresholding/threshd-configuration.xml", "etc/threshd-configuration.xml")
+                    .withFile("sentinel-thresholding/thresholds.xml", "etc/thresholds.xml")
+                    .build())
+            .withMinion()
+            .withSentinel()
+            .withTelemetryProcessing()
+            .withBlobStoreStrategy(BlobStoreStrategy.NEWTS_CASSANDRA)
+            .withJsonStoreStrategy(JsonStoreStrategy.POSTGRES)
+            .withIpcStrategy(IpcStrategy.KAFKA)
```

Comment:
Let's use JMS instead of Kafka for this one. Shouldn't matter for the test and will shave off almost 2GB of RAM.